src/wallet/rpcwallet.cpp

```diff
@@ -174,8 +174,12 @@ static void WalletTxToJSON(interfaces::Chain& chain, const CWalletTx& wtx, UniVa
     entry.pushKV("timereceived", (int64_t)wtx.nTimeReceived);
 
     // Add opt-in RBF status
-    const std::string rbfStatus = confirms <= 0 && SignalsOptInRBF(*wtx.tx) ? "yes" : "no";
-    entry.pushKV("bip125-replaceable", rbfStatus);
+    const bool signals_rbf = confirms <= 0 && SignalsOptInRBF(*wtx.tx);
+    entry.pushKV("replaceable", signals_rbf);
+    if (IsDeprecatedRPCEnabled("bip125")) {
```

Comment:
In 26c7fc0c38 _rpc: deprecate 'bip125-replaceable' in favour of 'replaceable'_

Could you grab this via the `Chain` interface, i.e. call `chain.rpcEnableDeprecated("bip125")` instead of moving `IsDeprecatedRPCEnabled()` to util? Or is it better to do it this way and take the opportunity to remove `rpcEnableDeprecated` by changing [this one](https://github.com/bitcoin/bitcoin/blob/26c7fc0c3879da366128195feff1e8b0112c35db/src/wallet/rpcwallet.cpp#L1756) too? (maybe scope creepish, sorry)